pyaedt/workflows/hfss/push_excitation_from_file.py

```diff
@@ -94,13 +94,24 @@ def callback():
     mainloop()
 
 
+aedt_process_id = None
 if "PYAEDT_SCRIPT_PORT" in os.environ and "PYAEDT_SCRIPT_VERSION" in os.environ:
     port = os.environ["PYAEDT_SCRIPT_PORT"]
     version = os.environ["PYAEDT_SCRIPT_VERSION"]
 else:
     port = 0
     version = "2023.2"
-with Desktop(new_desktop_session=False, close_on_exit=False, specified_version=version, port=port) as d:
+
+if os.getenv("PYAEDT_SCRIPT_PROCESS_ID", None):
```

Comment:
@Samuelopez-ansys  since this is repeated in a few files, why not have a function defined somehwere in pyaedt. 
e.g. `aedt_process_id = pyaedt.get_aedt_process_id()`